include/moveit_grasps/state_validity_callback.h

```diff
@@ -39,11 +39,19 @@
 #ifndef MOVEIT_GRASPS__STATE_VALIDITY_CALLBACK
```

Comment:
Linter won't like the double underscore.  I'd convert all these to `#pragma once` as all compiler support that.